File: bit-java01/src/main/java/step11/ex3/Test03.java
package step11.ex3;

public class Test03 {

  public static void main(String[] args) {
    // 1) Calculator3 설계도에 따라 메모리를 준비한다.
    Calculator3 calc = new Calculator3();
    
    // 2) Calculator3 설계도에 정의된 연산자를 사용하여 
    //    Calculator3 설계도에 따라 만든 메모리를 다룬다.
    calc.plus(10); // 수퍼 클래스 Calculator.plus()
    calc.plus(20); // 수퍼 클래스 Calculator.plus() 
    calc.minus(7); // 수퍼 클래스 Calculator.minus()
    calc.multiple(2); // 수퍼 클래스 Calculator2.multiple()
    calc.divide(3);  // 수퍼 클래스 Calculator2.divide()
    calc.mod(12); // 자신의 메서드 Calculator3.mod()
    
    System.out.println(calc.result);
    

  }

}

File: bit-python01/src08/test03.py
# 모듈 사용하기 - 함수에 별명 붙이기

# => from 모듈명 import 함수명 as 별명
from calculator import plus as p, minus
 
# => 별명이 붙은 함수는 별명으로 호출해야 한다. 
print(p(10, 20))

# => 별명이 붙은 함수는 원래 이름으로 호출할 수 없다.
#print(plus(10, 20)) # 오류!

# => 별명이 붙지 않은 함수는 원래 이름으로 호출한다.
print(minus(10, 20))
